src/Packagist/WebBundle/Controller/PackageController.php

```diff
@@ -142,47 +140,44 @@ public function fetchInfoAction(Request $req)
         $user = $this->getUser();
         $package->addMaintainer($user);
 
-        $response = array('status' => 'error', 'reason' => 'No data posted.');
```

Comment:
this is changing the response when the method is not called as POST. You will need to check for ``$form->isSubmitted()`` for invalid forms to change the response content to keep it